src/main/java/seedu/duke/common/Messages.java

```diff
@@ -24,18 +24,20 @@
     //Parser messages
     public static final String MESSAGE_MODULE_CODE_EMPTY = "Module not specified.";
     public static final String MESSAGE_INVALID_MODULE_CODE = "Invalid module code.";
-    
+
     public static final String MESSAGE_LESSON_FIELDS_EMPTY = "Missing lesson details.";
     public static final String MESSAGE_INVALID_LESSON_TYPE = "Invalid lesson type entered.";
     public static final String MESSAGE_INVALID_LESSON_LINK = "Invalid link entered.";
     public static final String MESSAGE_INVALID_LESSON_EMAIL = "Invalid email entered.";
-    
+
     public static final String MESSAGE_TASK_FIELDS_EMPTY = "Missing task details.";
     public static final String MESSAGE_INVALID_TASK_DEADLINE = "Invalid/missing deadline.";
-    
+
     public static final String MESSAGE_NON_INTEGER_INDICES = "Warning, non-integer values removed: %s";
     public static final String MESSAGE_OUT_OF_BOUNDS_INDICES = "Warning, out of bounds index removed: %s";
 
+    public static final String MESSAGE_INVALID_DAY_TIME = "Invalid format. Please enter a valid day and time!";
```

Comment:
I notice that this is the only file modified push request. Is this message in use?

Reply:
Ivan told me to add this to the messages class. He's gonna use it in the parser class